lib/Dialect/FIRRTL/FIRRTLTypes.cpp

```diff
@@ -1054,6 +1104,7 @@ llvm::Optional<int64_t> firrtl::getBitWidth(FIRRTLType type) {
             return llvm::Optional<int64_t>(None);
         })
         .Case<ClockType, ResetType, AsyncResetType>([](Type) { return 1; })
+        // TODO: RefType
```

Comment:
Any reason to not just add this?

Reply:
Uncertainty!  Not sure how these methods interact throughout the compiler ("What is this really asking") and was already reaching re:confidence in prototyping.  I'd guess here we want to report the bitwidth of the base type (only reasonable use I'd say is if width inference ends up invoking this?)?  FWIW I suspect this file is full of likely places we'll need to audit or touch to accommodate the new type, so maybe the scattered TODO's are misleading in a sense.